content/blog/2020-12-18-metadocencia/index.Rmarkdown

```diff
@@ -0,0 +1,110 @@
+---
+title: 'Building capacities to teach data science in Latin America'
+author:
+  - Yanina Bellini Saibene
+  - Paola Corrales
+  - Patricia Loto
+date: '2020-12-18'
+categories:
+  - learn
+description: |
+  The birth and growth of Metadocencia in Latin America
+  to nurture a community of Spanish-speaking educators in 2020 and beyond.
+slug: metadocencia
+photo:
+  author: Element5 Digital
+  url: https://unsplash.com/photos/OyCl7Y4y0Bk
+---
+
+## What?
+
+On March 16th, 2020, all the in-person classes and training opportunities were canceled in Argentina due to the pandemic with no clear indication on when they will resume. Hence a group of academics, researchers, and teachers with deep ties to the Latin American research and teaching community that have been working in the R community together co-founded [MetaDocencia](https://www.metadocencia.org/). 
+
+MetaDocencia means *MetaTeaching* in Spanish. Our mission *is to nurture a community of Spanish-speaking educators by teaching concrete, evidence-based, and student-centered educational methods.*
```

Comment:
I would remove the italics from the mission statement and the sentence below (line 25).